Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -367,9 +367,6 @@ private bool TryGenerateActivationPoint(IMixedRealityHand jointedHand, out Vecto
                     referenceJoint2 = TrackedHandJoint.Palm;
                     break;
                 case SolverSafeZone.RadialSide:
```

Comment:
I think you actually found a bug here! https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8896

Reply:
And going all the way back to the PR, `Ulnar` originally referred to different joints until the syntax was updated and the joints appear to have been copied incorrectly

https://github.com/microsoft/MixedRealityToolkit-Unity/commit/d07969ad624eb8c4682350e02bdf359841c9657d

![image](https://user-images.githubusercontent.com/3580640/120852279-d81ed980-c52e-11eb-9e8b-a0f8ff96ca60.png)

Reply:
Hmm.... I will need to check with the original author to see what the intent was.

Reply:
Confirmed with original author. The proper values for UlnarSide should be pinky and wrist. Updated them in the latest commit.